src/parsimmon.js

```diff
@@ -136,6 +151,8 @@ Parsimmon.Parser = (function() {
     var numParsers = parsers.length;
     if (numParsers === 0) return fail('zero alternates')
 
+    parsers.forEach(assertParser);
```

Comment:
err, oops i just merged this, but could you change these to not rely on `Array::forEach`? IE back-compatibility is really important for a number of parsimmon-using projects.